lib/lib/model/driver.dart

```diff
@@ -39,6 +39,6 @@ class DriverModel {
 
   String get getYearOfBirth => dateOfBirth.substring(0, 4);
 
-  String get getSearchableParams => "$getFullName $getYearOfBirth $nationality ";

```

Comment:
Itt több dolog is van: 

1, Az olyan paraméter nélküli gettereket mint ezek, nem szoktuk `get` -tel prefixelni. Egyrészt ez nagyon Java-s, másrészt nem függvények vannak itt, hanem propertyk, amiket ugyanúgy be lehet hivatkozni mint bármi más osztályszintű változót, nincs értelme külön nevezéktannal illetni őket. Ez a három property szebb lenne ha így volna elnevezve: 

```
   String get fullName => "$lastName $firstName";

  String get yearOfBirth => dateOfBirth.substring(0, 4);

  String get searchableParams => "$getFullName $getYearOfBirth $nationality ".toLowerCase();
```

2, A `searchableParams` típusa lehetne `List<String>`, mert úgy egyértelműbb lenne kívülről használva hogy 'ez egy olyan lista, aminek elemei a keresés során felhasználhatók', amit a String típus most nem sugall egyértelműen

3, A `searchableParams` -nak nem itt a helye. Annak eldöntése, hogy egy versenyző milyen paraméterek mentén kereshető, nem a model osztály felelősségi köre, hanem az üzleti logikáé. Ha ezt az elvet követjük, ennek az információnak a Service rétegben kellene előállni, nem a model osztályon